client/src/pages/tasks/Form.js

```diff
@@ -424,10 +429,15 @@ const BaseTaskForm = props => {
       "notes",
       "assessment_customFieldEnum1"
     )
-    task.responsibleOrg = utils.getReference(task.responsibleOrg)
     task.customFieldRef1 = utils.getReference(task.customFieldRef1)
     const { edit } = props
     const variables = { task: task }
+
+    // strip attendees fields not in data model
+    variables.task.taskedOrganizations = variables.task.taskedOrganizations.map(
```

Comment:
having to do this seems wrong. We can fix it in the future, but any better/safer way of doing this in the meanwhile?

Reply:
Why not use `utils.getReference(a)` instead of the `Object.without(…)` on the next line (as was done before)?